tests/data/cassettes/apps.epi.test_api/TestExposureApi.test_valid_requests.yaml

```diff
@@ -0,0 +1,73 @@
+interactions:
```

Comment:
@casey1173 you shoudn't need to create these, i think you should just be able to rename the existing.

For example, this one is `tests/data/cassettes/test_epi_api/TestExposureApi.test_valid_requests.yaml`

Reply:
thanks! I was wondering what was going on with the 11,000+ LOC added, I didn't even touch those files